cmd.go

```diff
@@ -116,13 +115,12 @@ func (b *Bot) executePassiveCommands(cmd *PassiveCmd) {
 		go func() {
 			defer wg.Done()
 
-			log.Println("Executing passive command: ", cmdName)
 			result, err := cmdFunc(cmd)
 			if err != nil {
 				log.Println(err)
 			} else {
 				mutex.Lock()
-				b.messageHandler(cmd.Channel, result, cmd.Nick)
+				handlers.Response(cmd.Channel, result, cmd.Nick)
 				mutex.Unlock()
```

Comment:
Don't know, IMO channels will just make the protocol implementation harder to do, because insteand of registering a simple func the protocols will need to have a for and keep reading on the channel, so AFAICT this is simpler.